docs/bugs.json

```diff
@@ -1,4 +1,15 @@
 [
+    {
+        "name": "ByteOptimizationBug",
```

Comment:
We never explicitly added the word "Bug". What about `ByteInstructionOptimization'?